### PR TITLE
Update peg

### PR DESCRIPTION
**Pull request by aidan-starke**
## Description

- Update Peg contract addresses for Mainnet & Kovan
- Add validators to withdraw proof

## Notes

- Withdrawals on new Peg contracts are inactive, meaning bridge is paused for withdrawals

### CODE REVIEW

**Review by eupston — APPROVED** [review]

**Comment by jordy25519** [comment]
kovan withdraw activated, need to update with new abi: https://github.com/cennznet/bridge-contracts/blob/main/abi/ERC20Peg.json

File: components/Withdraw.tsx [hunk 1]
```diff
@@ -133,7 +133,7 @@ const Withdraw: React.FC<{}> = () => {
           await api.rpc.ethy.getEventProof(eventProofId)
         ).toJSON();
         if (versionedEventProof !== null) {
-          eventProof = versionedEventProof.EventProof;
+          eventProof = versionedEventProof.eventProof;
```
Comment (jordy25519) [inline components/Withdraw.tsx]: I think the case is required to convert properly
```suggestion
          eventProof = versionedEventProof.EventProof;
```

Comment (aidan-starke) [reply components/Withdraw.tsx]: not sure why this has changed, I had to make this same change in [this PR](https://github.com/cennznet/bridge-ui/pull/43/commits/dad99e29535af2b9defe63dce4049b1425e476f8) in order for it to work - using the UI with `versionedEventProof.EventProof` returns eventProof as undefined

Comment (jordy25519) [reply components/Withdraw.tsx]: I think that PR using cennznet/api 2.x.x version. in 1.5.x still lowercase. caused by polkadot/api changes in between the versions

Comment (jordy25519) [reply components/Withdraw.tsx]: wrote this comment on the wrong PR several times 🤦‍♀️

Comment (aidan-starke) [reply components/Withdraw.tsx]: ahh yup forgot to reset yarn

**Pull request merged**